_posts/george_mihaiescu/2017-01-23-How-we-use-ELK-and-Ntopng-to-track-and-visualize-data-downloads.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "How we use ELK (Elasticsearch, Kibana, Logstash) and Ntopng to track and visualize data downloads"
+breadcrumb: true
+author: george_mihaiescu
+date: 2017-01-23
+categories: george_mihaiescu
+tags:
+    - OpenStack
+    - Ceph
+    - Collaboratory
+teaser:
+    info: How we use ELK (Elasticsearch, Kibana, Logstash) and Ntopng to track and visualize data downloads
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+## Introduction
+<a href="http://www.cancercollaboratory.org/">Cancer Genome Collaboratory</a> is a cloud environment built using open-source technologies that aims to offer cancer researchers access to compute resources close to the large data sets used in their analysis. 
+
+Currently we store more than 540 TB of data and have available 2500 CPU cores deployed on top of Openstack and Ceph mainly, but we also use a number of other open-source tools to manage and monitor the infrastructure.
```

Comment:
In this sentence you mix collab current usage (data) and available cores. Should we have total storage and total CPU instead ?
"Collaboratory infrastructure uses OpenStack and Ceph at its core, to provide the research community with xx PB of storage and 2500 CPU cores, but the team also relies on a number of other open-source tools to manage and monitor the infrastructure."